src/Components/Body.jsx

```diff
@@ -1,28 +1,32 @@
 import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
 
 import { useState, useEffect } from 'react';
 import { getArticleById } from '../api';
 
 const Body = ()=>{
 const {article_id}= useParams()
 const [article, setArticle] = useState([]);
-// console.log(article)
+const [IsLoading,setIsLoading]= useState(false)
+
 
 useEffect(() => {
+  setIsLoading(true)
     getArticleById(article_id).then(({ article }) => {
-        console.log(article)
+    setIsLoading(false)
       setArticle(article);
     });
   }, []);
 
-  if (!article) return <p>Loading...</p>;
+  if (IsLoading) return <p>Loading...</p>;
 
   return (
     <div>
       <h1>{article.title}</h1>
       <p>{article.body}</p>
       <p>Author: {article.author}</p>
       <p>Topic: {article.topic}</p>
+      <Link to={`/articles/${article_id}/comments`}>View Comments</Link>
```

Comment:
Instead of a link you could render the comments component here if you want them to appear underneath the same article :)